src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,151 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 1 << 4;
```

Comment:
Could you please provide an exact value for this expression instead? It seems quite confusing